data.sql

```diff
@@ -11,4 +11,33 @@ INSERT INTO animals (id,name, date_of_birth,escape_attempts,neutered,weight_kg,s
     (8,'Angemon', '2005-06-12', 1, true, -45, '');
     (9,'Boarmon', '2005-06-07', 7, true, 20.4, '');
     (10,'Blossom', '1998-09-13', 3, true, 17, '');
-    (11,'Ditto', '2022-05-14', 4, true, 22, '');
\ No newline at end of file
+    (11,'Ditto', '2022-05-14', 4, true, 22, '');
+
+INSERT INTO owners (id, full_name, age)
+VALUES (1, 'Sam Smith', 34),
+  (2, 'Jennifier Orwell', 19)
+  (3, 'Bob', 45)
+  (4, 'Melody Pond', 77)
+  (5, 'Dean Winchester', 14),
+  (6, 'Jodie Whittaker', 38)
+
+INSERT INTO species (id, name)
```

Comment:
- [ ]  [OPTIONAL] You don't need to add an id here as it will be auto-assigned. You need to add name as per the requirements. I highly recommend you update it.